reactor-core/build.gradle

```diff
@@ -75,12 +75,13 @@ dependencies {
 	//Optional JDK 9 Converter
 	jsr166backport "io.projectreactor:jsr166:$jsr166BackportVersion"
 
-	testCompile "junit:junit:$jUnitVersion"
-	testCompile "org.junit.jupiter:junit-jupiter-api:${jUnitJupiterVersion}"
-	testCompile "org.junit.platform:junit-platform-launcher:1.6.1"
-	testCompile "org.junit.jupiter:junit-jupiter-params:${jUnitJupiterVersion}"
-	testRuntime "org.junit.jupiter:junit-jupiter-engine:${jUnitJupiterVersion}"
-	testRuntime "org.junit.vintage:junit-vintage-engine:${jUnitJupiterVersion}"
+	implementation platform("org.junit:junit-bom:${jUnitPlatformVersion}")
+	testCompile "junit:junit:${jUnitVersion}"
```

Comment:
IIRRC `junit-bom` also manages `junit:junit`, could you please check it?

Reply:
I checked that. It does not manage it directly, only transitively through `-vintage` (and uses 4.13 that we want). The problem is that there are other dependencies in the build that introduce 4.12 and this is what is being picked up.

Reply:
Either 4.12 and 4.13 would work, so I'd rather remove one more property to manage :)

Reply:
Like I said, I left it for a reason: we want 4.13, because we compile against `org.junit.function.ThrowingRunnable` which is in 4.13